e2e/cypress/integration/enterprise/system_console/channel_moderation_spec.js

```diff
@@ -794,7 +794,7 @@ describe('Channel Moderation Test', () => {
         it('Effect of changing System Schemes on a Channel for which Channel Moderation Settings was never modified', () => {
```

Comment:
Have seen failing tests but not related to this PR.
![Screen Shot 2020-03-27 at 2 41 36 PM](https://user-images.githubusercontent.com/5334504/77729684-3c483280-703a-11ea-8923-3623045df0fb.png)